journal-of-medical-internet-research.csl

```diff
@@ -218,6 +238,16 @@
       </if>
     </choose>
   </macro>
+  <macro name="webcite">
+    <choose>
+      <if type="webpage post-weblog" match="any">
+        <group delimiter=": " prefix=". ">
+          <text value="archived at" text-case="capitalize-first"/>
+          <text variable="note"/>
```

Comment:
Is there any other way of solving this? Can users manually somehow add the webcite link then? usually people don't cite many website anyway, so it wouldn't be the majority of entries anyway.